src/collaborative/ot/ot.ts

```diff
@@ -61,12 +65,30 @@ const transformations: {
  */
 export function transform(
   toTransform: CoreCommand,
-  executed: CoreCommand
+  executed: CoreCommand,
+  coreGetters: CoreGetters
```

Comment:
transformation cannot depend on getters because they should not depend on the current state of the spreadsheet. Different users can be in different states (before everyone have received every revision). So different states would mean different transformation, and ultimately, all clients would not converge to the same state.